assets/js/common/Tags/Tags.jsx

```diff
@@ -15,13 +16,33 @@ const tagValidationDefaultMessage = (
   </>
 );
 
+function TagDeleteButton({ onClick, disabled }) {
```

Comment:
I find strange the `TagDeleteButton`. I would call it `ExistingTag` maybe.

Reply:
done